calico-cloud/threat/web-application-firewall.mdx

```diff
@@ -167,7 +167,7 @@ spec:
   dataSet: waf
   period: 1m
   lookback: 1h
-  query: '"rule_info" IN {"*942100*"}'
+  query: '"rules.id" = "942100"'
```

Comment:
This `query` may not be valid. I have made [a comment in the original PR](https://github.com/tigera/docs/pull/821#discussion_r1290591083). @bmckercher123  Do you know where this change comes from?

Reply:
> But all that might be redundant because we just merged code to remove the need for the global alert to trigger waf events.

This is still in our docs/next and it is public. If [this step](https://docs.tigera.io/calico-enterprise/next/threat/web-application-firewall#create-global-alerts) is not needed anymore, it is better to remove it from our doc. People will see it and our test cycles will run it which fails atm.

Reply:
This is still in our public doc/next and people will see it. Our test cycle need to validate the doc steps which is failing atm.